libraries/AP_PiccoloCAN/piccolo_protocol/ECUPackets.c

```diff
@@ -603,7 +603,7 @@ int decodeECU_TelemetrySlow1PacketStructure(const void* _pg_pkt, ECU_TelemetrySl
 
     // Input voltage in Volts
     // Range of voltage is 0.0f to 25.5f.
-    _pg_user->voltage = float32ScaledFrom1UnsignedBytes(_pg_data, &_pg_byteindex, 0.0f, 1.0f/10.0f);
```

Comment:
This is a generated file, should not be modified

Reply:
I have reverted the changes to the piccolo_protocol.